services/brig/src/Brig/Provider/API.hs

```diff
@@ -1001,6 +1041,11 @@ botDeleteSelf bid cid = do
 --------------------------------------------------------------------------------
 -- Utilities
 
+checkAllowed :: Maybe UserId -> (Handler r) ()
```

Comment:
```suggestion
-- | If second factor auth is enabled, make sure that end-points that don't support it, but should, are blocked completely.  (This is a workaround until we have 2FA for those end-points as well.)
guardSecondFactorDisabled :: Maybe UserId -> Handler r ()
```

this is 3 suggestions, some or all of which you may want to ignore:
- comment
- rename
- remove parens in type signature